snap/snapcraft.yaml

```diff
@@ -0,0 +1,299 @@
+name: freecad-realthunder
+base: core20
+adopt-info: freecad
+summary: An open source parametric 3D CAD modeler (realthunder's version)
+description: |
+  FreeCAD is a parametric 3D modeler. Parametric modeling
+  allows you to easily modify your design by going back into
+  your model history and changing its parameters. FreeCAD is
+  open source (LGPL license) and completely modular, allowing
+  for very advanced extension and customization.
+
+  FreeCAD is multiplatfom, and reads and writes many open
+  file formats such as STEP, IGES, STL and others.
+
+  This snap packages the FreeCAD fork of realthunder 
+  (https://github.com/realthunder/FreeCAD/) + the Assembly3 workbench
+  (https://github.com/realthunder/FreeCAD_assembly3).
+
+  Do _not_ complain to upstream about issues with this snap!
+
+  Commands:
+    freecad-realthunder:      Run FreeCAD
+    freecad-realthunder.cmd:  Run FreeCAD command line interface
+    freecad-realthunder.pip:  Install python packages for user (not system-wide). 
+
+grade: devel
```

Comment:
This is a safeguard for development versions; it prevents accidentally pushing a build from this to the `stable` channel in snapcraft. In the stable branch, we must switch this to stable. If you don't care, you could also use stable in LinkDaily.